tests/integration/ovs_test.py

```diff
@@ -17,20 +17,34 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
+from contextlib import contextmanager
 import pytest
```

Comment:
contextlib and time are part of the stdlib but pytest is not, therefore they should be in their own block before pytest.

Reply:
Done!